metadata-ingestion/src/datahub/ingestion/source/aws/dynamodb.py

```diff
@@ -174,8 +174,9 @@ def populate_attribute_definitions(self, table) -> Dict[str, str]:
                 attribute_definitions[attr] = new_value
         else:
             table_name = table["TableName"]
-            logger.warn(f"Table {table_name} streaming is NOT enabled.")
+            logger.error(f"Table {table_name} streaming is NOT enabled.")
             self.report.report_table_streaming_disabled(table_name)
```

Comment:
I suppose there is no point to this anymore as originally this was used to report issues at the end of the run. So we can get rid of this entirely.